ros/src/computing/planning/motion/packages/astar_planner/nodes/velocity_set/libvelocity_set.h

```diff
@@ -233,13 +240,59 @@ class ObstacleTracker
 
   double calcVelocity(const geometry_msgs::Point& cpos)
   {
-    tf::Vector3 cx;
-    tf::pointMsgToTF(cpos, cx);
-    double dx = (position_buf_[1]-cx).length()-(position_buf_[0]-cx).length();
-    ros::Duration dt = time_buf_[1]-time_buf_[0];
-    kf_.predict();
-    double v = kf_.update(dx/dt.toSec());
-    v = (v > moving_thres_) ? v : 0.0;
+    double v;
+    ros::Duration dt = time_buf_[1]-time_buf_[0]; double dx;
```

Comment:
double dx; => into new line

Reply:
This is just for debug, I will change it when removing the debug print's